ui/imports/utils/Utils.qml

```diff
@@ -186,9 +193,7 @@ QtObject {
     }
 
     function countWords(text) {
-        if (text.trim() === "")
-            return 0;
-        return text.trim().replace(/  +/g, " ").split(" ").length;
+        return parseMnemonicWords(text).length
```

Comment:
`countWords` is very generic method and in general should not depend on more specific `parseMnemonicWords` which may potentially change in a way unexpected for generic method like `countWords.

I think we could just rename `parseMnemonicWords` to `splitWords` because it's the actual logic of that function.

Reply:
Indeed, I didn't think of this.
It's just that this method is only used in one place, related to mnemonic:
https://github.com/status-im/status-desktop/blob/5068f85bc17f7cbf073a20206886c25ca3567e69/ui/imports/utils/Utils.qml#L199-L201

That's why I didn't think it's a generic function 😄 
But you're right, I will rename it.